src/cmd/factory/impl/common.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/alcionai/corso/src/pkg/account"
 	"github.com/alcionai/corso/src/pkg/backup/details"
 	"github.com/alcionai/corso/src/pkg/control"
+	ctrlTD "github.com/alcionai/corso/src/pkg/control/testdata"
```

Comment:
Should we just use `testdata` here (same thing in a few other files).

Reply:
I waffled on that here, and can certainly use the default.  For info, I feel there's now a number of `foo/testdata` packages in play now, so the likelihood of collision is higher.  But I should probably stick to the unaliased version until needed.